src/Microsoft.CodeQuality.Analyzers/Core/QualityGuidelines/DoNotRaiseExceptionsInExceptionClauses.cs

```diff
@@ -19,8 +19,10 @@ namespace Microsoft.CodeQuality.Analyzers.QualityGuidelines
     /// inside a filter block and there is no language representation for fault blocks in either language.
     /// So this analyzer just checks for throw statements inside finally blocks.
     /// </remarks>
-    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
```

Comment:
same here.

Reply:
I don't want to comment out the analyzer completely as I would need to add comments to the test code referencing this type. We should revert these changes soon, so I will keep it as is for now.